### ISSUE TITLE
terminate called after throwing an instance of 'kaguya::LuaRuntimeError'

### ISSUE THREAD

**Issue by astridx** [systemed/tilemaker#448]
I have got the  file [rueber2.osm.pbf.txt](https://github.com/systemed/tilemaker/files/10087174/rueber2.osm.pbf.txt). I created it via 

` $ osmium extract --bbox=7.356202,50.281992,7.402227,50.307076 --set-bounds --strategy=smart //europe-latest.osm.pbf  --output /mnt/hgfs/Downloads/osm/rueber2.osm.pbf`

Know I want to create vector tiles via

`$ ./tilemaker --input /rueber.osm.pbf --output rueber --process ./resources/process-openmaptiles.lua --config ./resources/config-openmaptiles.json clipIncompleteEntities=true`

But it is not possible.

```
$ ./tilemaker --input /rueber2.osm.pbf --output rueber.osm --process ./resources/process-openmaptiles.lua --config ./resources/config-openmaptiles.json clipIncompleteEntities=true
Layer place (z0-14)
Layer boundary (z0-14)
Layer water (z6-14)
Layer waterway (z8-14)
Layer ocean (z0-14) -> water
Bounding box 7.3562, 50.282, 7.40223, 50.3071
Reading .shp ocean
Sorting loaded shapes
Reading .pbf /mnt/hgfs/Downloads/osm/rueber2.osm.pbf
Block 0/4 ways 0 relations 0        
Sorting nodes
terminate called after throwing an instance of 'kaguya::LuaRuntimeError'
  what():  attempt to call a nil value
Abgebrochen (Speicherabzug geschrieben)

```

It would be greate, if you could tell me, what I am making wrong.

**Comment by systemed** [comment]
`clipIncompleteEntities=true` is an Osmosis option, it's not something that tilemaker understands.

That apart, if I run your command with that .pbf, it works fine for me. Does it work for you if you remove the `clipIncompleteEntities=true` bit? Are you using the latest version of tilemaker from source or an earlier version?

**Comment by astridx** [comment]
Thank you for your very quick reply.
`clipIncompleteEntities=true` I had used with osium. I had copied that here incorrectly. Sorry.
I am using the version of Tilemaker that is here in the master branch of this repo. 
I have now narrowed down my error further. Because it worked for you, I replaced my config with one that is here in the repo. That works. I guess the error is in my config then. When I know what exactly I did wrong, I'll paste it here in case it may be of interest to others as well.
